scalableminds/webknossos
# Always aim to compute the full (ad-hoc) mesh

This PR removes the hard mesh chunk limit and instead introduces a soft throttle mechanism to avoid putting full load for the mesh generation for a single segment (e.g., a huge segment could take forever to load and should not cause a tremendous load on client + backend).

### URL of deployed dev instance (used for testing):
- https://fullmesh.webknossos.xyz/

### Steps to test:
- load some ad-hoc meshes for big segments
- I tested it with this segment: https://fullmesh.webknossos.xyz/datasets/sample_organization/l4_sample/view#3584,3789,1369,0,1.3
- I also set a breakpoint at the the `sleep` statement to ensure that it was triggered

### Issues:
- fixes https://github.com/scalableminds/webknossos/issues/4835

------
(Please delete unneeded items, merge only when none are left open)
